__tests__/app.test.js

```diff
@@ -256,3 +256,61 @@ describe("GET /api/articles", () => {
     expect(res.body.message).toBe("Route not found");
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  const newComment = {
+    username: "rogersop",
+    body: "A very nice and lovely comment!",
+  };
+
+  test("status:201, responds with park newly added to the database", async () => {
+    const article_id = 1;
+    const { body } = await request(app)
+      .post(`/api/articles/${article_id}/comments`)
+      .send(newComment)
+      .expect(201);
+    expect(body.comment).toEqual({
+      comment_id: 19,
+      article_id: 1,
+      author: "rogersop",
+      body: "A very nice and lovely comment!",
+      votes: 0,
+      created_at: expect.any(String),
+    });
+  });
+  test("404: shows correct error status code and message when given valid 'id type' that does not exist in database", async () => {
+    const article_id = 9999;
+    const { body } = await request(app)
+      .post(`/api/articles/${article_id}/comments`)
```

Comment:
I'd argue that, as it's only being used once, it would be DRYer to just pass your 9999 straight into your .post endpoint, rather than saving it as a variable.